---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  fig.path = "man/figures/README-",
  out.width = "100%"
)
```

# davizr

<!-- badges: start -->

<!-- badges: end -->

*D*ifferential *A*nalysis *VIS*ualiz*ER*

`davizr` is an R package that makes it easy to import and visualize data for sequencing data differential analysis such as identifying differentially expressed genes (DEGs) from RNA-seq data, calling differentially accessible regions (DARs) from ATAC-seq data, etc.

## Installation

You can install the development version of davizr from [GitHub](https://github.com/) with:

```r
# install.packages("devtools")
devtools::install_github("trev-f/davizr")
```

## Development

Pull the repository from GitHub:

```sh
git pull utia-gc/davizr
```

Navigate into the repository and install development dependencies using `renv`:

```r
renv::install()
```

I recommend only using the `DESCRIPTION` file to maintain package state.
`renv` should only be used for environment isolation.
Don't use `renv::snapshot()` to maintain package state in the `renv.lock` file.

## Example

A common task in RNA-seq analysis is simply to construct a matrix of read counts within genes from a collection of files generated by `featureCounts`.
