---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  fig.path = "man/figures/README-",
  out.width = "100%"
)
```

# davizr

<!-- badges: start -->

<!-- badges: end -->

*D*ifferential *A*nalysis *VIS*ualiz*ER*

`davizr` is an R package that makes it easy to import and visualize data for sequencing data differential analysis such as identifying differentially expressed genes (DEGs) from RNA-seq data, calling differentially accessible regions (DARs) from ATAC-seq data, etc.

## Installation

You can install the development version of davizr from [GitHub](https://github.com/) with:

``` r
# install.packages("devtools")
devtools::install_github("trev-f/davizr")
```


## Contribute

Contributions are welcome to this open source project.

To contribute, fork this repo and clone it locally.
After making changes, open a pull request against the upstream repo, `utia-gc/davizr`.
We will review it and merge any changes or leave feedback.

### Development dependencies

We recommend using `renv` to manage dependencies.

To get started, first remove any traces of an existing `renv` project, then initialize a new project:

``` r
renv::deactivate(clean = TRUE)
renv::init(bioconductor = TRUE)
```

When initializing the project, select option `1` for explicit mode to only install libraries declared in the DESCRIPTION file.
